packages/wonder-blocks-dropdown/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -62,7 +62,6 @@ exports[`wonder-blocks-dropdown example 1 1`] = `
       onTouchCancel={[Function]}
       onTouchEnd={[Function]}
       onTouchStart={[Function]}
-      open={false}
```

Comment:
The `open` prop wasn't being destructured so it was being included when the sharedProps were spread into this button.  Note: each `aria-expanded` should not correspond to whatever the value `open` was in the same button.